content/ux-guide/our-approach/defining-design.md

```diff
@@ -0,0 +1,82 @@
+
+
+## Design supports people's needs
+
+We proactively engage with people to understand their needs and determine how well our decisions have met those needs. Since some of our work involves creating solutions that are used, managed, delivered, or informed by government employees or partner organizations, who we engage in the work may not always be just members of the public.
+
+Stakeholders are anyone with a legitimate interest in what we’re designing. Stakeholders include our agency partners and the people they serve—that is, their constituents. We regularly collaborate with our agency partners, and while all design processes must take their needs into consideration, our process ultimately puts those most impacted by our work at the center of our decision-making as our primary concern.
+
+## Design should be cross-functional
+
+When design is practiced together, it creates shared understanding. We bring together cross-functional teams, including a range of different types of designers, to arrive at more well-informed, less [biased]({{ '../../research/bias/' | url }}) decisions. Outside of the range of methodologies that come from different design approaches, cross-functional teams allow for two kinds of thinking:
+- **Divergent thinking** helps us to identify opportunities for decision making, and to explore possible choices
+- **Convergent thinking** helps us narrow our choices, and follow through on our decisions
+
+Intentionally transitioning between divergence and convergence helps make the decision-making process visible and participatory for team members.
+
+
+## Design shapes organizations and vice-versa
+
+Products and services impose requirements on the organizations that support them, and vice-versa. We shouldn't design an agency’s homepage to include images on every news post, for example,  if we haven’t yet made certain that the agency can reasonably source images for each of its news posts. To best support the endeavor as a whole, we want to proactively acknowledge when our work may require nw investments, organizational shifts or new resources.
+
+
+## Design is never done
+
+We don’t have all the answers. And that’s okay. We take an iterative, cyclical, participatory, [lean]({{ '../../our-approach/stay-lean' | url }}) approach. We try things out and learn from them. We commit to continuous improvement using critiques, [wireframes](https://methods.18f.gov/make/wireframing/), [prototypes](https://methods.18f.gov/make/prototyping/), [usability tests](https://methods.18f.gov/usability-testing), service blueprints, etc. and we strive to build the capacity of our partners to do the same. 
+
+{% image_with_class "assets/ux-guide/iterative-design.png"  "A conceptual image that describes iterative design. The phrase 'iterative design' is surrounded by four labels connected by a circle of clockwise-pointing arrows. Starting at the top, the labels read 'concept', 'try it out', 'gather feedback', 'incorporate feedback" %}
```

Comment:
`image_with_class` takes 3 arguments in the order `<file>`  `<class>` `<alt-text>`, so currently the alt text is being used as the class names. You'll need to add a `""` before the current alt text to make it the right argument. I can add this to the documentation.